_data/BraedenNorris/BraedenNorris.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Braeden Norris # No longer than 18 characters
+institution: Lyons Township High School # no longer than 58 characters
+profile_pic: braeden.jpg # Name and extension of your profile picture(ex. mona.png)
```

Comment:
Please correct the image file name.

Reply:
I've uploaded a properly sized image and the name matches the md file.